README.md

```diff
@@ -60,6 +60,23 @@ $ cat > ./packages/api-server/rate-limit-config.json <<EOF
 EOF
 ```
 
+### Fix wrong data
```

Comment:
can we move fix wrong data into a new README.md under cli folder? this is a one-time fix, no need to be in the main README.md?

Reply:
Updated